app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -14,17 +14,24 @@ import androidx.appcompat.app.AppCompatActivity
 import androidx.coordinatorlayout.widget.CoordinatorLayout
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 
-
 class DrawActivity : AppCompatActivity() {
 
     private val viewModel: DrawViewModel by viewModels()
     private val dialogUtils = DialogUtils
+    private val animUtils = AnimationUtils
     private var isAddDialogOpen = false
     private var isEditDialogOpen = false
     private var isDeleteDialogOpen = false
     private var isLayerListViewSheetOpen = false
     private var isColorPickerSheetOpen = false
-    lateinit var behavior: BottomSheetBehavior<LinearLayout>
+    private lateinit var behavior: BottomSheetBehavior<LinearLayout>
```

Comment:
1. I don't think these would ever need to be lateinit. You're just giving them resource ids, not views. Lateinit isn't something that's very commonly needed.
2. Make this a map. Better to have 1 variable than a ton of them. Then you wouldn't even need the list because the map could be iterated.

Reply:
maps are cool, set this up this way.